pact/Pact/Core/SizeOf.hs

```diff
@@ -367,11 +452,23 @@ instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (DefPact n t b i)
 instance (SizeOf t, SizeOf i) => SizeOf (DefSchema t i)
 instance (SizeOf n, SizeOf i) => SizeOf (DefTable n i)
 instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Def n t b i)
+
 instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Module n t b i)
+
 instance (SizeOf t, SizeOf i) => SizeOf (IfDefun t i)
 instance (SizeOf t, SizeOf i) => SizeOf (IfDefPact t i)
 instance (SizeOf n, SizeOf t, SizeOf i) => SizeOf (IfDefCap n t i)
 instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (IfDef n t b i)
 instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Interface n t b i)
 
 instance SizeOf Namespace
+
+
+type Bytes = Word64
+
+newtype SizeOfByteLimit
```

Comment:
Can we move these closer to the top of the file? These definitions are pretty important.